models/prices/prices_tokens.sql

```diff
@@ -125,6 +125,7 @@ VALUES
     ("frax-frax", "avalanche_c", "FRAX", "0xd24c2ad096400b6fbcd2ad8b24e7acbc21a1da64", 18),
     ("fxs-frax-share", "avalanche_c", "FXS", "0x214db107654ff987ad859f34125307783fc8e387", 18),
     ("avax-avalanche", "avalanche_c", "WAVAX", "0xB31f66AA3C1e785363F0875A1B74E27b85FD66c7", 18),
+    ("avax-avalanche", "avalanche_c", "AVAX", "0x0000000000000000000000000000000000000000", 18),
```

Comment:
@RomualdH for native tokens to a blockchain, the intention is to use the prices from lines 15-48. it's a different join to prices.usd in downstream models. for example (pseudo code):
```
left join prices.usd
on source.block_time = prices.minute
and blockchain is null
and symbol = 'AVAX'
```

Reply:
notice the address is left out of the join

Reply:
looks like avax/bnb are taken care of, maybe add matic to lines above?